internal/scheduler/leaderelection.go

```diff
@@ -5,6 +5,9 @@ import (
 	"fmt"
```

Comment:
I'm planning to remove this file in a subsequent PR as we plan to do leader election via K8s rather than by database